source/compatibility.tex

```diff
@@ -1161,6 +1127,10 @@
 Valid \CppIII{} code that assumes that global \tcode{operator new} only
 throws \tcode{std::bad_alloc} may execute differently in this International
 Standard.
+Valid \CppIII{} code that replaces the global replaceable \tcode{operator new}
+is ill-formed in this International Standard,
+because the exception specification of \tcode{throw(std::bad_alloc)}
+was removed.
```

Comment:
Add a cross-reference to the Annex C entry describing the removal?

Reply:
This is the Annex C entry describing the removal.

Reply:
Oh, sorry, I somehow thought this should refer to the removal of dynamic exception specifications or of `throw()`, but you're right, that's not what matters here.